portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,241 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  font-family: 'Open Sans', sans-serif;
+}
+body {
+  padding: 0;
+  margin: 0;
+}
+html {
+  scroll-behavior: smooth;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+/*--- navigation bar ---*/
+h1 {
+    background-color: #84bdff;
+    padding: 0 10px 0 10px;
+}
+.navbar {
+	background: #79b8ff;
+}
+
+.nav-link,
+.navbar-brand {
+	color: #fff;
+    font-family: 'Open Sans', sans-serif;
+	cursor: pointer;
+}
+.icon1{
+    height: 50%;
+    width: 47px;
+}
+
+.icon2{
+    height: 50%;
+    width: 30px;
+}
+
+.nav-link {
+	margin-right: 1em !important;
+}
+
+.nav-link:hover {
+	color: #000;
+}
+
+.navbar-collapse {
+	justify-content: flex-end;
+}
+
+/*--- header ---*/
+
+.header {
+	background-color: #79b8ff;
+	background-size: cover;
+	background-position: center;
+	position: relative;
+}
+
+.overlay {
+	position: absolute;
+	min-height: 100%;
+	min-width: 100%;
+	left: 0;
+	top: 0;
+	background: rgba(0, 0, 0, 0.6);
+}
+
+.description {
+	left: 50%;
+	position: absolute;
+	top: 45%;
+	transform: translate(-50%, -55%);
+	text-align: center;
+}
+
+.description h1 {
+	color: #494949;
+    font-family: 'Open Sans', sans-serif;
+
+}
+
+.description a {
+	color: #fff;
+	font-size: 1.3rem;
+    font-family: 'Open Sans', sans-serif;
```

Comment:
nit: Style guide suggests merging elements together into shorthand whenever possible (in this case, you should be able to fit all font stuff into a single 'font' attribute